src/Pages/Dictionary.js

```diff
@@ -1,10 +1,38 @@
-import React from 'react';
-import WordTermDisplay from '../Components/WordTermDisplay';
+import React, { useState, useEffect } from 'react';
+import WordTerm from '../Components/WordTerm';
 
-function Dictionary() {
-  return (
-    <WordTermDisplay />
-  );
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function WordTermDisplay() {
```

Comment:
I think it would be good to still name the page Dictionary. This might cause errors with the routing, and is less explanatory!